Realm/RLMFindOptions.mm

```diff
@@ -0,0 +1,100 @@
+////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2020 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+#import "RLMFindOptions.h"
+#import "RLMFindOptions_Private.hpp"
+#import "RLMBSON_Private.hpp"
+
+@interface RLMFindOptions() {
+    realm::app::RemoteMongoCollection::RemoteFindOptions _options;
+};
+@end
+
+@implementation RLMFindOptions
+
+- (instancetype)initWithLimit:(NSInteger)limit
+                   projection:(id<RLMBSON> _Nullable)projection
+                         sort:(id<RLMBSON> _Nullable)sort {
+    if (self = [super init]) {
+        [self setProjection:projection];
```

Comment:
Using property setter syntax (i.e. `self.projection = projection`) is more idiomatic.